cds/webapi/C#/BlazorCDS/BlazorCDS/wwwroot/appsettings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "AzureAd": {
+    "Authority": "https://login.microsoftonline.com/organizations",
+    "ClientId": "52470569-ea16-4732-aacf-8bb93ee709b9",
```

Comment:
I don't think we should include this ClientId.
The instructions to run this sample should explain how they can register their app and put their own clientid.
We could use this as a placeholder: `11111111-1111-1111-1111-111111111111`

Unfortunately, I don't think the globally approved clientid we use for other samples will work in this case because it needs to have a specific callback URI.

Reply:
replaced with placeholder GUID. However, haven't updated the instructions yet. I referenced the article in readme which basically tells to follow the steps listed there and create a new Blazor WASM project with that client id